test/linux-generic/pktio_ipc/pktio_ipc1.c

```diff
@@ -309,8 +313,24 @@ int main(int argc, char *argv[])
 		exit(EXIT_FAILURE);
 	}
 
+	odp_cpumask_default_worker(&mask, 0);
+	cpu = odp_cpumask_first(&mask);
+
+
+	CPU_ZERO(&cpu_set);
+	CPU_SET(cpu, &cpu_set);
+
+	pid = getpid();
+
+	if (sched_setaffinity(pid, sizeof(cpu_set_t), &cpu_set)) {
```

Comment:
odph_odpthread_getaffinity and  opdh_odpthread_setaffinity can not be used here because it now multi threaded example, i.e. there is no any thread.  And that is test/linux-generic/ example. I will send v2 with updated current code as it more simple.